test/workflow/Workflow.test.js

```diff
@@ -9,6 +9,25 @@ import { Fetcher } from '../../src/index.js';
 describe('Workflow', function () {
   this.timeout(30 * 1000);
 
+  it('should run the deep crawler correctly', async () => {
+    const wf = fox
+      .init('https://pokemondb.net/pokedex/national')
```

Comment:
add `.config({ cache: testCache() })`, and then add `@fast` to the test description so it will run in pre-commit and be fast